string NAME=is_location
bool ERASE_WHEN_FALSE=0

ObjectLocationID location_id

bool truth_value
